integrations/elasticsearch-integration.mdx

```diff
@@ -1,127 +1,144 @@
 ---
 title: "Elasticsearch"
 ---
-Collects Elasticsearch performance and health metrics from Elasticsearch clusters and nodes.
 
-Prerequisites
--------------
-<ul>
-    <li>Middleware Host agent should be already installed on your machine, in order to use Elasticsearch integration.</li>
-</ul>
+# Overview
 
-## Setup
-1. You will have to create a yaml file in your system containing database collection details, as follows:
+The Elasticsearch Integration allows users to track their Elasticsearch performance and health metrics from Elasticsearch clusters and nodes. Use our default dashboard to understand system-wide usage patterns and identify performance bottlenecks. Drill into all of your data within the Elastic ecosystem.
 
-Ex. `/home/ubuntu/elasticsearch-creds.yaml`
-```
+# Prerequisites
+
+1. Middleware Host agent must be installed on your machine
+
+
+# Setup
+
+### Step 1: Create YAML file
+
+Create a `.yaml` file in your system with the rest of your database collection details
+
+`/home/ubuntu/elasticsearch-creds.yaml`
+
+### Step 2: Add Elasticsearch Credentials to the YAML file
+
+Ensure you have added your `username` and `password` to the YAML file
```

Comment:
Does this need to be an admin user / what permissions are required by this user? I'd normally say something like the following:

"Add credentials for the user which you'd like Middleware to connect as. This user should be [an admin/have the following permissions]"

Reply:
pretty sure they are just a users elasaticsearch credentials